### PR TITLE
make wu faster

### PR DESCRIPTION
**Pull request by envyaims**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by envyaims** [comment]
References https://github.com/cpinitiative/usaco-guide/issues/3795

File: solutions/silver/cf-1017D.mdx [hunk 1]
```diff
@@ -9,15 +9,15 @@ author: Chongtian Ma
 
 ## Alternative Solution
 
-We can first precompute the wu values for each pair of strings. Then, we can prefix sum up the number of strings that gives each wu value of each string (represented by `pref_wu` in the following implementation). Thus, for each query we can utilize binary search to acquire the greatest prefix that have wu value not greater than $k$.
+We can first precompute the wu values for each pair of strings. Then, we can prefix sum up the number of strings that gives each wu value of each string (represented by `pref_wu` in the following implementation). Thus, for each query we can utilize output our prefix sum array.
```
Comment (ryanchou-dev) [inline solutions/silver/cf-1017D.mdx:12]:
```suggestion
We can first precompute the wu values for each pair of strings. Then, we can sum up the number of strings that gives each wu value of each string in a prefix sum (represented by `pref_wu` in the following implementation). Thus, for each query, we can utilize the output of our prefix sum array.
```

**Review by ryanchou-dev — APPROVED** [review]
merry christmas envy <3

**Comment by envyaims** [comment]
> merry christmas envy <3

omg i forgot you're now a CM too orz 🙏

File: solutions/silver/cf-1017D.mdx [hunk 2]
```diff
@@ -27,6 +27,8 @@ minus wu of $(i \oplus j)$ will also give wu values of the bits that are the sam
 #include <bits/stdc++.h>
 using namespace std;
 
+const int max_k = 100;
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1017D.mdx:30]: make uppercase

Comment (envyaims) [reply solutions/silver/cf-1017D.mdx:30]: bruh

**Pull request merged**